docs/2.7.0/docs/canton/architecture/domains/domains.rst

```diff
@@ -284,7 +284,7 @@ ranging from a relational database to a distributed blockchain. Components can
 be shared among integrations, for example, a mediator implemented on a
 relational database can be used together with a blockchain-based sequencer.
 
-.. toctree::
+.. .. toctree::
```

Comment:
@yves-da Will this cause any issue for Canton?

Reply:
Yes, it will.

But it can be fixed relatively easily.

Reply:
I'll prepare PR for canton which prepares for this change.